support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -140,10 +140,13 @@ export const tests: Tests = {
 	adFreeTierThree: {
 		variants: [
 			{
-				id: 'control',
+				id: 'control', // Tier2 add-free
```

Comment:
Minor nitpick:

```suggestion
				id: 'control', // Tier2 ad-free
```

Reply:
My spelling is not the best, all updated, thanks